### DISCUSSION TITLE
Support for ATmega32u4 boards?

### DISCUSSION THREAD

**Discussion by smsnkaty** [shanteacontrols/OpenDeck#238] [Q&A] [answered]
I've been working with another MIDI software on my KEYESTUDIO Pro Micro ATmega32u4 5V board. I'd like to jump over to OpenDeck, but I can't seem to find the right package to download and flash. [This early post](https://shanteacontrols.com/2017/12/14/opendeck-on-arduino/) makes it look like the early OpenDeck branded boards used ATmega32u4 MCUs, making it easy to adapt to Arduino Pro Micros. I believe the newer OpenDeck boards are based on RPI2040.

Is there still a way for me to get this running on my ATmega32u4? Thanks

**Comment by paradajz** [comment, marked as answer]
ATmega32u4 isn't supported for a very long time now. Recommended alternative is Raspberry Pi Pico which is both cheaper and simpler to flash.

**Reply by smsnkaty** [reply]
Sounds good. I also just ordered an OpenDeck M board, so I'm sure I'll be back with questions. Thanks for your help.
